app/views/layouts/minimal.html.erb

```diff
@@ -0,0 +1 @@
+<!-- Intentionally left blank -->
```

Comment:
Maybe explain why this is intentionally left blank?

Reply:
It's now renamed to "Empty".  Sometimes we would prefer an empty layout rather than the application layout.
Sessions Controller: `layout 'empty', :only => [ :timeout_check ]`
Home Controller: `layout 'empty', :only => [ :extend_session_iframe ]`

The application layout has a bunch of files being required such as `<%= render "layouts/timeout" if current_user %>` that we do not need when performing this XMLHttpRequest.